napalm_yang/parsers/xml.py

```diff
@@ -7,6 +7,15 @@
 class XMLParser(BaseParser):
 
     @classmethod
+    def init_native(cls, native, device_output=[]):
+        device_trans = []
+        for r in device_output:
+            if isinstance(r, dict) and all([isinstance(x, (str, unicode)) for x in r.values()]):
+                # Junos returns commands enclosed by a key
+                device_trans.append("\n".join(r.values()))
+        return super(XMLParser, cls).init_native(native, device_trans)
```

Comment:
Use the super that you import from future, mostly for py3 compatibility. See `napalm_yang/translators/text.py` for an example.